ui/app/src/views/MigrateView.tsx

```diff
@@ -148,18 +152,27 @@ function MigrateView() {
             {migrateMutation.error.message}
           </Alert>
         )}
-        {migrateMutation.isSuccess && (
-          <Stack direction={'row'} spacing={1}>
+        {!isLoading && data !== undefined && data !== null && migrateMutation.isSuccess && (
```

Comment:
`{!isLoading && data && migrateMutation.isSuccess && (`

Reply:
I am following the [Perses UI guidelines](https://github.com/perses/perses/blob/main/ui/ui-guidelines.md#additional-best-practices) here: 

> Use explicit `null` or `undefined` checks instead of relying on JS falsiness